### PR TITLE
Update instances

### PR DESCRIPTION
**Pull request by gabtorre**

### CODE REVIEW

File: content/departments/support/tools/index.md [hunk 1]
```diff
@@ -41,26 +41,44 @@ Read more about working with managed instances [here](../process/support-managed
...
+    {
+        "location": "top",
+        "message": "This instance is down for maintenance."
+    },
+```
+
 - [Test instance deployed with Docker Compose](https://cse-aws-test.sgdev.org/) ([_management doc_](../process/managing-cs-aws.md))
 - [Test instance deployed with Kubernetes](https://cse-k8s.sgdev.org/) ([_management doc_](../process/managing-cs-k8s.md))
-- [Single Docker for local testing](https://docs.sourcegraph.com/admin/install/docker)
+- [Sourcegraph on Single-container Docker for local testing](https://docs.sourcegraph.com/admin/install/docker)
```
Comment (DaedalusG) [inline content/departments/support/tools/index.md:61]:
```suggestion
#### Local Sourcegraph Instances
- [Sourcegraph on Single-container Docker for local testing](https://docs.sourcegraph.com/admin/install/docker)
```

File: content/departments/support/tools/index.md [hunk 2]
```diff
@@ -41,26 +41,44 @@ Read more about working with managed instances [here](../process/support-managed
 
 There are some test environments created specifically for the CS team to run tests on when helping customers. They allow the team to reproduce and debug issues without affecting our production instances.
 
+#### Sourcegraph Instances
```
Comment (DaedalusG) [inline content/departments/support/tools/index.md]:
```suggestion
#### External DNS Sourcegraph Instances
```

**Review by DaedalusG — APPROVED** [review]
This all looks good to me -- some minor comments for organizational suggestions

**Comment by northyg** [comment]
Nice edition! Thanks for doing this :)

**Review by northyg — APPROVED** [review]

File: content/departments/support/tools/index.md [hunk 3]
```diff
@@ -41,26 +41,44 @@ Read more about working with managed instances [here](../process/support-managed
...
+
+Add a maintenance [notice](https://docs.sourcegraph.com/admin/config/settings#notices) to the Global settings if you are testing, experimenting or upgrading [cse-k8s](https://cse-k8s.sgdev.org) or [cse-aws-test](https://cse-aws-test.sgdev.org).
```
Comment (DaedalusG) [inline content/departments/support/tools/index.md:46]: We may want to amend the text of the notice such that it doesn't explicitly say the instance is down. The AER testing could also note what they're doing as far as testing in the banner.

**Pull request merged**